Web/starter_project_barbies/components/team/TeamMemberCard.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactNode } from "react";
+import {
+  BsFacebook,
+  BsLinkedin,
+  BsInstagram,
+  BsEmojiSmileUpsideDown,
+} from 'react-icons/bs'
+import Image from 'next/image'
+import { TeamMember } from '@/types/team'
+
+export type TeamMemberCardProps = {
```

Comment:
If you've already defined its type and if your not planning on adding another property to this, you can just use the TeamMember type you imported. Theres no need to define another type here.